src/containers.js

```diff
@@ -313,7 +271,7 @@ const ChatStackChild = new Lang.Class({
                                                   '',
                                                   GObject.ParamFlags.READWRITE |
                                                   GObject.ParamFlags.CONSTRUCT_ONLY,
-                                                  TriggerableEventQueue.$gtype),
+                                                  Queue.TriggerableEventQueue.$gtype),
```

Comment:
Probably not - we want to specify that it is a queue of some sort. The duplication in the name is not ideal, but `Queue.TriggerableEvent` doesn't suggests that it is an event emitter and not a queue.